src/software/backend/input/network/networking/network_client.cpp

```diff
@@ -192,10 +192,13 @@ void NetworkClient::filterAndPublishVisionData(SSL_WrapperPacket packet)
 
 void NetworkClient::filterAndPublishGameControllerData(Referee packet)
 {
-    RefboxGameState game_state = network_filter.getRefboxGameState(packet);
-    world.updateRefboxGameState(game_state);
+    if (world.field().isFieldValid())
```

Comment:
Didn't realize `isFieldValid` already existed. This is a super-nit, but while we're here can we rename to just `isValid()`? The `Field` part is redundant.

Reply:
done